### PR TITLE
Fixed error in Dockerfile.reva

### PR DESCRIPTION
**Pull request by JarCz**
There is an error when building docker image from Dockerfile.reva:
"cp: cannot stat '/go/src/github/cs3org/reva/cmd/reva/reva': No such file or directory"

### CODE REVIEW

File: Dockerfile.reva [hunk 1]
```diff
@@ -20,5 +20,5 @@ FROM golang:1.13
 
 WORKDIR /go/src/github/cs3org/reva
 COPY . .
-RUN make build-reva-docker && cp /go/src/github/cs3org/reva/cmd/reva/reva /go/bin/reva
+RUN make build-reva-docker && cp /go/src/github/cs3org/reva/cmd/revad/reva /go/bin/reva
```
Comment (labkode) [inline Dockerfile.reva]: What is the problem you are trying to solve? This Dockerfile is for the client cli tool (reva) and not for the server (revad)

**Comment by labkode** [comment]
I got why you wanted to that, but is better if you fix it here:

```
 build-reva-docker: off
-       go build -o ./cmd/revad/reva ./cmd/reva
+       go build -o ./cmd/reva/reva ./cmd/reva
```

Thanks!

File: Dockerfile.reva (hunk 1 above)
Comment (JarCz) [reply Dockerfile.reva]: Fixed error build docker image from command line, 
issue: https://github.com/cs3org/reva/issues/530

After fix, build om localmachine works at commands:

```
docker build -t reva -f Dockerfile.reva .
``` 
 or
 
```
git clone https://github.com/JarCz/reva.git
cd reva
cp Dockerfile.reva Dockerfile
docker build -t reva .
```

**Comment by labkode** [comment]
@JarCz please do the changes I've requested in the Makefile:

```
 build-reva-docker: off
-       go build -o ./cmd/revad/reva ./cmd/reva
+       go build -o ./cmd/reva/reva ./cmd/reva
```

and I will merge this PR, the Makefile is the place that makes sense to fix it.

**Comment by JarCz** [comment]
I changed Makefile. Please merge repositories.

**Pull request merged**